lib/developer_portal/app/controllers/developer_portal/admin/accounts_controller.rb

```diff
@@ -2,7 +2,7 @@ class DeveloperPortal::Admin::AccountsController < DeveloperPortal::BaseControll
 
   liquify prefix: 'account'
 
-  authorize_resource
+  authorize_resource class: Account
```

Comment:
This wasn't needed before 🤔 Better to investigate why `resource_class` is no longer `Account` by default in 2.3.0.

Reply:
Because this is what it used to do in 1.7.1 to know the name:
https://github.com/CanCanCommunity/cancancan/blob/c9afa64a9290bdbbb5e2b5f2eaa91cea9894118a/lib/cancan/controller_resource.rb#L254-L258
And this is what it does in 2.3:
https://github.com/CanCanCommunity/cancancan/blob/1760baab0214e4deb5142b1993adaaddb48cec60/lib/cancan/controller_resource_name_finder.rb#L9-L11

They don't do the same, one tries to constantize `DeveloperPortal::admin::Account` which raises NameError and then returns the name, and the other one tries to constantize `DeveloperPortal::Admin::Account` and can do it so it does not return the "|| name" which would be the Account.

Reply:
Hmm. I know why.

https://github.com/CanCanCommunity/cancancan/blob/c9afa64a9290bdbbb5e2b5f2eaa91cea9894118a/lib/cancan/controller_resource.rb#L254-L258

```ruby
[namespace, name.camelize].join('::')
=> "developer_portal::admin::Account"

[namespace, name.camelize].join('::').singularize.camelize.constantize
=> NameError: wrong constant name admin

name
=> "account"
```

Luck.

Reply:
Sorry, @Martouta. I hand't seen your comment. I guess we were both investigating. Good to know we got the same conclusion 😊